app/js/newTest.js

```diff
@@ -0,0 +1,23 @@
+let testForm = document.querySelector('#testForm')
+
+testForm.addEventListener('submit', async function(e) {
+    e.preventDefault()
+    let inputLength = document.querySelector('#testName').value.length
+    if (inputLength > 0 && inputLength < 256) {
+        let testData = {}
+        testData.name = testForm.testName.value
+        testData = await jsonToFormData(testData)
+        const testPath = "test"
```

Comment:
does this need to be a const? only using it once

Reply:
changed to let